src/Controller/AboutController.php

```diff
@@ -270,17 +273,19 @@ public function peopleAction(Request $request, string $type) : Response
                 $editorInChief = $leadership->filter(function (Person $person) {
                     return 'Editor-in-Chief' === $person->getTypeLabel();
                 });
-                $foundingEditorInChief = $leadership->filter(function (Person $person) {
-                    return 'Founding Editor-in-Chief' === $person->getTypeLabel();
-                });
+                $foundingEditorInChief = $people->get(self::FOUNDING_EDITOR_IN_CHIEF_ID)
+                    ->then(function (Person $person) {
+                        return new ArraySequence([$person]);
+                    })
+                    ->otherwise($this->softFailure('Failed to load the Founding Editor-in-Chief', new EmptySequence()));
                 $deputyEditors = $leadership->filter(function (Person $person) {
-                    return !in_array($person->getTypeLabel(), ['Editor-in-Chief', 'Founding Editor-in-Chief']);
+                    return 'Editor-in-Chief' !== $person->getTypeLabel();
```

Comment:
I will deploy this tomorrow and change the data just prior to deployment so that we don't get a duplicate of Randy's profile on the leadership page.

Reply:
The data has already been updated. Randy is BRE now so he is not appearing on the leadership page until this is deployed.